testproject/Assets/Tests/Runtime/RpcTestsAutomated.cs

```diff
@@ -0,0 +1,207 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using TestProject.ManualTests;
+using MLAPI.RuntimeTests;
+using MLAPI;
+using Debug = UnityEngine.Debug;
+
+namespace TestProject.RuntimeTests
+{
+    public class RPCTestsAutomated
+    {
+        private bool m_TimedOut;
+
+        private int m_MaxFrames;
+
+        private GameObject m_PlayerPrefab;
+
+        private int m_OriginalTargetFrameRate;
+
+        [SetUp]
+        public void SetUp()
+        {
+            // Just always track the current target frame rate (will be re-applied upon TearDown)
+            m_OriginalTargetFrameRate = Application.targetFrameRate;
+
+            // Since we use frame count as a metric, we need to assure it runs at a "common update rate"
+            // between platforms (i.e. Ubuntu seems to run at much higher FPS when set to -1)
+            if (Application.targetFrameRate < 0 || Application.targetFrameRate > 120)
+            {
+                Application.targetFrameRate = 120;
+            }
```

Comment:
I'd suggest 60 frames as it is more common but I'd be OK with 120 too for sure, pick the one you'd prefer.

Reply:
Will stick with 120 as that has been passing with flying colors and seems to yield "consistent results" across all platforms.